### DISCUSSION TITLE
Menambahkan fitur toast

### DISCUSSION THREAD

**Discussion by muchfahmiarif** [sensasi-apps/sensasi-pos#116] [Ideas]
Om @sensasi-delight saya ingin menambahkan fitur toast jika produk berhasil/gagal ditambahkan, tidak hanya fungsi toast bisa banyak. Penggunaan toast agar lebih interaktif dan mempermudah aksesibilitas

**Comment by zainphp** [comment]
dear @muchfahmiarif,

terima kasih atas masukannya, maka dari itu terciptalah isu #117
